tests/test_utilsEfd.py

```diff
@@ -0,0 +1,153 @@
+# This file is part of cp_pipe.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import unittest
+
+import lsst.utils.tests
+
+from lsst.cp.pipe.utilsEfd import CpEfdClient
+
+
+class UtilsEfdTestCase(lsst.utils.tests.TestCase):
+    """Unit test for EFD access code."""
+
+    def setUp(self):
+        # Initialize a client.
+        try:
+            self.client = CpEfdClient()
+        except Exception as e:
+            self.log.warning(f"Could not initialize EFD client: {e}")
```

Comment:
I don't think we warn on skipped tests.  I don't know if it's possible to initialize the client at the main level and use skip directives ... what does Merlin do for summit utils tests?

Reply:
Looks like a raise that skips the test in setup: https://github.com/lsst-sitcom/summit_utils/blob/main/tests/test_efdUtils.py#L67  I've switched to do the same.